Resources/Prototypes/_NF/Maps/debug.yml

```diff
@@ -11,4 +11,20 @@
           mapNameTemplate: "DevNF"
         - type: StationJobs
           availableJobs:
+            Contractor: [ -1, -1 ]
+            Pilot: [ -1, -1 ]
+            Mercenary: [ -1, -1 ]
             StationRepresentative: [ -1, -1 ]
+            StationTrafficController: [ -1, -1 ]
+            SecurityGuard: [ -1, -1 ]
+            Valet: [ -1, -1 ]
+            Janitor: [ -1, -1 ]
+            MailCarrier: [ -1, -1 ]
+            Sheriff: [ -1, -1 ]
+            Bailiff: [ -1, -1 ]
+            SeniorOfficer: [ -1, -1 ]
+            Brigmedic: [ -1, -1 ]
+            DetectiveNF: [ -1, -1 ]
+            Deputy: [ -1, -1 ]
+            Cadet: [ -1, -1 ]
+            Lawyer: [ -1, -1 ]
```

Comment:
There's no spawn points for these jobs, right?  How does that work?  Also, I know roundstart joining does some weird stuff with overflow jobs, and since these are all overflow jobs, how's that handled?

Reply:
There is still late join